cmd/main.go

```diff
@@ -101,7 +101,8 @@ func findCurrentRepo() (string, error) {
 			err = fmt.Errorf("%s", string(exitErr.Stderr))
 		}
 		// give up, let the user figure it out
-		return "", fmt.Errorf("could not determine repository path from module data, package data, or by initializing a module: %v", err)
+		return "", fmt.Errorf("could not determine repository"+
```

Comment:
Done

Reply:
Go against go fmt

Reply:
`go fmt` removes this space?

Reply:
Yes, you can try locally. 
You add the space and then run go fmt ./...

Reply:
Then just mark as resolved all the suggestions of adding spaces before `+`. Probably that looks nicer to me because in Python you are forced to use them.